.ci/java-versions.properties

```diff
@@ -5,7 +5,7 @@
 # are 'java' or 'openjdk' followed by the major release number.
 
 ES_BUILD_JAVA=openjdk12
-ES_RUNTIME_JAVA=java11
+ES_RUNTIME_JAVA=adoptopenjdk11
```

Comment:
This will be reverted before merging. The purpose here is to get a PR CI run before merging.